pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -10,10 +10,12 @@ showSyntax() {
   echo "               [{--switch-generation | -G} generation] [--verbose...] [-v...]" >&2
   echo "               [-Q] [{--max-jobs | -j} number] [--cores number] [--dry-run]" >&2
   echo "               [--keep-going] [-k] [--keep-failed] [-K] [--fallback] [--show-trace]" >&2
```

Comment:
```suggestion
  echo "               [{--keep-going, -k} {--keep-failed, -K} [--fallback] [--show-trace]" >&2
```

Reply:
You mean `[{--keep-going | -k}] [{--keep-failed | -K}]`, right?

Reply:
Actually, it seems like `-k` and `-K` only work for `nix-build` rather than `nix build`, so best not to change this

Reply:
see: https://github.com/NixOS/nix/issues/2105

Reply:
Well, that they are mutually exclusive does not mean that they always can be used.
The usage description does not detail which options may be used with which actions.

Similar would be the case for `--max-jobs` and `-j`, BTW.

Reply:
The syntax we're using for `[{ --long, -l }]` I'm understanding as equivalent flags, so I would rather just leave it as is

Reply:
If compatibility is a concern between `nix build` and `nix-build`, then I suggest removing the usage description for the shorthands, to deprecate them. Only the long options can be used with both build commands.

Reply:
I'm happy to just leave them there as is and it actually matches `nixos-rebuild --help`

Reply:
`nixos-rebuild --help` says
```
[{--keep-failed | -K}] [{--keep-going | -k}]
```
so I guess that means "no revert"?

To "leave them there as is" is ambiguous, meaning leave the change as is, or leaving the codebase as is.

Reply:
Resolve the thread(s) if you are happy with what is.

Reply:
I just checked `nixos-rebuild --help` and I see:

```
[--cores  number] [--keep-going | -k] [--keep-failed | -K]
```

So I think let's match that

Reply:
Done